service/controller/v15/resource/cloudformation/delete.go

```diff
@@ -31,8 +31,21 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 			return microerror.Mask(err)
 		}
 
+		stackName := aws.String(key.MainGuestStackName(customObject))
+
+		r.logger.LogCtx(ctx, "level", "debug", "message", "disable protection guest main")
```

Comment:
Log message reads a bit strangely to me. Having `guest main stack` in the message would give me more context. 

Also not sure this is aligned with the updated fmt docs.

https://github.com/giantswarm/fmt/blob/master/go/log_messages.md#ensuring-state

Reply:
I have removed those. There were only for testing